core/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
```

Comment:
Just thought of this: do we need this new `core` module?

We have a root src tree in the parent module under `google-container-tools-intellij/src...` where presumably this could live?

Reply:
That actually was my initial intention, but this introduces circular dependencies between root project and subproject unfortunately since root pulls in all sub-modules for the build.

Reply:
ah you're right of course. sg.